resources/definitions/GEEETECH_A10T.def.json

```diff
@@ -0,0 +1,59 @@
+{
+    "version": 2,
+    "name": "GEEETECH A10T",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Amit L",
+        "manufacturer": "GEEETECH TECH.",
+        "file_formats": "text/x-gcode",
+        "has_materials": true,
+        "machine_extruder_trains":
+        {
+            "0": "GEEETECH_A10T_1",
+            "1": "GEEETECH_A10T_2",
+            "2": "GEEETECH_A10T_3"
+        }
+
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "GEEETECH A10T" },
+        "machine_width": {
+            "default_value": 220
+        },
+        "machine_height": {
+            "default_value": 220
+        },
+        "machine_depth": {
+            "default_value": 220

```

Comment:
In the [specifications on their website](https://www.geeetech.com/geeetech-a10t-3-in-1-out-mixcolor-auto-leveling-3d-printer-p-1107.html) they claim a build volume of 220x220x250. Did you make it smaller intentionally or was that a typo?